README.md

```diff
@@ -39,9 +39,10 @@ To further configure your plugin, use the following custom parameters in your `s
-| `subscribeToHttpApiLogs`    | Enable automatic subscription of the Datadog Forwarder to Http-api log groups. Defaults to `true`.                                                                                                                                                                                                                                                                                                                   |
-| `subscribeToWebsocketLogs`  | Enable automatic subscription of the Datadog Forwarder to Websocket log groups. Defaults to `true`.                                                                                                                                                                                                                                                                                                                  |
+| `subscribeToApiGatewayLogs` | Enable automatic subscription of the Datadog Forwarder to API Gateway Access log groups. Defaults to `true`.                                                                                                                                                                                                                                                                                                         |
+| `subscribeToHttpApiLogs`    | Enable automatic subscription of the Datadog Forwarder to Http-api Access log groups. Defaults to `true`.                                                                                                                                                                                                                                                                                                            |
+| `subscribeToWebsocketLogs`  | Enable automatic subscription of the Datadog Forwarder to Websocket Acess log groups. Defaults to `true`.                                                                                                                                                                                                                                                                                                            |
+| `subscribeToExecutionLogs`  | Enable automatic subscription of the Datadog Forwarder to Execution log groups for REST and Websocket APIs if subscriptions to their access logs are also enabled. Note: Cloudformation does not maintain these log groups. If the stack is removed the log groups will need to be removed manually or the next deploy may be unsuccessful. Defaults to `false`.                                                     |
```

Comment:
I'm a bit confused by how these four options work:
- It seems that for each type of API, you can choose separately whether or not you want to subscribe to access log groups. For HTTP and Websocket APIs, the parameter name matches the type of API (there are `subscribeToHTTPApiLogs` and `subscribeToWebsocketLogs` options). There is another parameter called  `subscribeToApiGatewayLogs` -- however, HTTP APIs and Websocket APIs are just types of API Gateway APIs. So, does this parameter override the other two? Or is it for REST APIs, which is the third type of API that doesn't seem to have its own setting?
- The setting for execution logs says that it it subscribes to execution logs for REST APIs and Websocket APIs. What about HTTP APIs, do they not have execution logs? Also, it says you need to have subscriptions to their access logs enabled as well -- why is that?
- Another thought: do customers want to have control over which exact types of APIs they want to subscribe to? E.g. do customers want logs from their HTTP APIs but NOT their Websocket APIs? Or would it be simpler to just have one option? This is more of a product question.

Reply:
- I can change API Gateway to Rest access logs to remove the confusion. 
- I should definitely replace the requirement for access logs to be enabled for execution logging to get a subscription. I was using that validation to confirm REST or Websocket logging was enabled, but it would be better to grab that from the provider section of the serverless config instead.
- Http APIs do not support execution logging currently so it is not included.
-  I think the plugin should have this much flexibility. The API Gateways already require additional setup in the `serverless.yml` at both the provider and function level. My understanding is these log groups can be very noisy and they may only be enabled for debugging, which means more flexibility is better if there is only an issue in one log group. @DarcyRaynerDD do you agree?

Reply:
I'm also thinking we should split the Execution log group env var into separate variables to give more control to the user. Having all or nothing for execution log groups seems troublesome since they will generate a lot of logs.